byterun/sys.c

```diff
@@ -619,7 +618,7 @@ CAMLprim value caml_sys_isatty(value chan)
 
   fd = (Channel(chan))->fd;
 #ifdef _WIN32
-  ret = Val_bool(_isatty(fd));
+  ret = Val_bool(caml_win32_isatty(fd));
         /* https://msdn.microsoft.com/en-us/library/f4s0ddew.aspx */
```

Comment:
It looks like this URL is no longer relevant; why not delete the comment?

Reply:
Done.